Find a functionally equivalent code:com.android.documentsui.RootsCache.updatePackageAsync:COMMENT
Method Modifier: public      
Comment:/**
 * Gather roots from storage providers belonging to given package name.
 */

@@ -1,6 +1,3 @@
 {
-    // Need at least first load, since we're going to be using previously
-    // cached values for non-matching packages.
-    waitForFirstLoad();
     new UpdateTask(packageName).executeOnExecutor(AsyncTask.THREAD_POOL_EXECUTOR);
 }

